testing/testcafe/tests/dataGrid/headerFilter.ts

```diff
@@ -27,3 +29,26 @@ test('The header filter should fit inside the viewport if the grid is scrolled h
   height: 400,
   headerFilter: { visible: true },
 }));
+
+test('HeaderFilter popup screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getHeaders().getHeaderRow(0).getHeaderCell(0).getFilterIcon())
+    // act
+    .expect(await takeScreenshot('header-filter-popup', dataGrid.element))
```

Comment:
It's better to add the extension to the screenshot name:

```suggestion
    .expect(await takeScreenshot('header-filter-popup.png', dataGrid.element))
```